src/controllers/HistoryController.ts

```diff
@@ -44,19 +46,23 @@ export class HistoryController {
     response.send(html);
   };
 
-  private latestHistory = async (request: Request, response: Response): Promise<void> => {
-    const records = await this.fetcherHistoryRepository.latest();
+  private latestHistory = async (_request: Request, response: Response): Promise<void> => {
```

Comment:
why `_`?

Reply:
it is to indicate that it is not used, but I can revert it in case you prefer.

If I would choose I would just write `_` alone for this case.

Reply:
I would prefer `_` if you want to indicate that, but if `_` prefix is some standard for not used variables, then let's leave your change.